components/button/theme.css

```diff
@@ -34,9 +114,9 @@
   white-space: nowrap;
   vertical-align: middle;
 
-  .children {
+ /* .children {
     padding-top: var(--button-children-padding-correction);
-  }
+  }*/
```

Comment:
**Q** Can we just not get rid of this block if it's commented out?

Reply:
Sorry, I have these comments removed in the next PR: https://github.com/teamleadercrm/teamleader-ui/pull/47/files